src/index.js

```diff
@@ -141,3 +86,12 @@ button.addEventListener('click', () => {
   titletextbox.value = '';
   render();
 });
+
```

Comment:
- [ ] According to project requirements, I kindly suggest that you add status update functionality.
- add methods related to the status updates (completed: true / false) in a new file and import it as a module here

Reply:
Instead of complete true or false I used 'doneBox' which implements as well a strikethrough on the task on the HTML, on the array changes from true/false depending if checked or unchecked.
if an edit task is necessary only need to press enter or switch fields to be recorded